frontend/src/data/variables/VariableProvider.ts

```diff
@@ -42,6 +44,56 @@ abstract class VariableProvider {
     return this.getDataInternal(datasets, breakdowns);
   }
 
+  filterByGeo(df: IDataFrame, breakdowns: Breakdowns): IDataFrame {
+    const fipsColumn: string =
+      breakdowns.geography === "county" ? "county_fips" : "state_fips";
+
+    if (breakdowns.filterFips !== undefined) {
+      const fips = breakdowns.filterFips as Fips;
+      if (fips.isState() && breakdowns.geography === "county") {
+        df = df.where((row) => fips.isParentOf(row["county_fips"]));
```

Comment:
It's generally good practice to avoid re-assigning function arguments (https://medium.com/the-andela-way/its-bad-practice-to-reassign-function-params-c37f289a5dc4). Usually doesn't matter but probably best to use a new variable. (same below)